apps/www/components/Article/metadata.js

```diff
@@ -72,6 +72,26 @@ const getJSONLDs = (meta) => {
   }
 }
 
+function getCitationMetaData(meta) {
+  const authors = meta.contributors
+    .filter((contributor) => contributor?.kind?.includes('Text'))
+    .map((contributor) => {
+      const parts = contributor.name.split(' ')
+      // Convert the name into the following format `Surname, Given Name(s)`
+      return `${parts[parts.length - 1]}, ${parts
+        .slice(0, parts.length - 1)
+        .join(' ')}`
```

Comment:
```suggestion
      return contributor.name
```

Since some people have multiple word names etc. I'd just give it as is.

There are other examples without the comma split:
https://www.w3.org/WAI/RD/wiki/Indexing